install-dev/upgrade/sql/1.7.0.0.sql

```diff
@@ -19,17 +25,11 @@ ALTER TABLE PREFIX_customer ADD `reset_password_validity` datetime DEFAULT NULL;
 ALTER TABLE PREFIX_employee ADD `reset_password_token` varchar(40) DEFAULT NULL;
```

Comment:
Hello @martinfojtik 
Just add  **'PREFIX_employee'** :smile: but with " ` " i can make it because, it's interpreted like code ahah